app/src/main/kotlin/com/wire/android/ui/authentication/create/details/CreateAccountDetailsScreen.kt

```diff
@@ -127,174 +125,179 @@ private fun DetailsContent(
+                item {
+                    WirePasswordTextField(
+                        value = state.password,
+                        onValueChange = onPasswordChange,
+                        labelMandatoryIcon = true,
+                        descriptionText = stringResource(R.string.create_account_details_password_description),
+                        imeAction = ImeAction.Next,
+                        autofillTypes = listOf(),
+                        modifier = Modifier
+                            .padding(horizontal = MaterialTheme.wireDimensions.spacing16x)
+                            .testTag("password"),
+                        state = if (state.error is CreateAccountDetailsViewState.DetailsError.TextFieldError.InvalidPasswordError) {
+                            WireTextFieldState.Error()
+                        } else {
+                            WireTextFieldState.Default
+                        }
+                    )
+                }
+
+                item {
+                    WirePasswordTextField(
+                        value = state.confirmPassword,
+                        onValueChange = onConfirmPasswordChange,
+                        labelText = stringResource(R.string.create_account_details_confirm_password_label),
+                        labelMandatoryIcon = true,
+                        imeAction = ImeAction.Done,
+                        keyboardActions = KeyboardActions(onDone = { keyboardController?.hide() }),
+                        autofillTypes = listOf(),
+                        modifier = Modifier
+                            .padding(
+                                horizontal = MaterialTheme.wireDimensions.spacing16x,
+                                vertical = MaterialTheme.wireDimensions.spacing16x
+                            )
+                            .testTag("confirmPassword"),
+                        state = if (state.error is CreateAccountDetailsViewState.DetailsError.TextFieldError) when (state.error) {
+                            CreateAccountDetailsViewState.DetailsError.TextFieldError.PasswordsNotMatchingError ->
+                                WireTextFieldState.Error(stringResource(id = R.string.create_account_details_password_not_matching_error))
+
+                            CreateAccountDetailsViewState.DetailsError.TextFieldError.InvalidPasswordError ->
+                                WireTextFieldState.Error(stringResource(id = R.string.create_account_details_password_error))
+                        } else WireTextFieldState.Default
                     )
                 }
             }
+            LaunchedEffect(Unit) {
+                focusRequesterFirstName.requestFocus()
+                keyboardController?.show()
+            }
+
+            WirePrimaryButton(
+                modifier = Modifier
+                    .padding(MaterialTheme.wireDimensions.spacing16x)
+                    .fillMaxWidth()
+                    .imePadding(),
```

Comment:
This `imePadding` also breaks this screen on Android 9:

https://github.com/wireapp/wire-android-reloaded/assets/30429749/e4c87141-9d57-4b20-8ddb-11d633feb4e1

Reply:
Same